content/en/policy-controller/overview.md

```diff
@@ -119,6 +116,7 @@ spec:
 
 Each `key` authority can contain these properties:
 - `key.data`: specifies the plain text string of the public key
+- `key.hashAlgorithm` (optional): specifies the signature digest for the key, e.g. `sha512`, `sha256`, `sha384` or `sha224`
```

Comment:
since this is optional, might want to mention the default.

Reply:
yes, I removed 
https://docs.sigstore.dev/policy-controller/overview#deprecated-policy-controller-validation-behavior